src/components/user-auth/registrationForm/RegistrationForm.jsx

```diff
@@ -371,3 +278,60 @@ const RegistrationForm = () => {
 }
 
 export default RegistrationForm
+
+const useStyles = makeStyles()(() => {
+  return {
+    title: {
+      fontSize: 32,
+      fontWeight: 600,
+      lineHeight: '40px',
```

Comment:
if it's not a big deal, plz, follow agreement we discussed - use numbers instead of strings with px ('40px')